Tasks/AzureFileCopy/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 74
+    "Patch": 75
```

Comment:
they are unequal in current master as well. I've just incremented the patch numbers of task.json and task.loc.json by 1 each from the current master.